src/shared/components/annotation/OncoKB.tsx

```diff
@@ -78,7 +78,15 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
             <span className={`${annotationStyles["annotation-item"]}`} />
         );
 
-        if (this.props.indicator)
+        if (this.props.indicator === null) {
+            // null means there is an error...
+            oncoKbContent = this.errorIcon();
```

Comment:
Actually, we have status in remote data (`MobXPromise`), but we are only passing the `result` field (the actual data part) to the components.

Maybe we should redesign our components so that they can be aware of the `status` of the data? I guess that is out of the scope of this hotfix PR...